theseus_gui/src/components/ui/ModpackVersionModal.vue

```diff
@@ -0,0 +1,341 @@
+                .filter((value, index, self) => self.indexOf(value) === index)
+            "
+            :multiple="true"
+            :searchable="true"
+            :show-no-results="false"
+            :close-on-select="false"
+            :clear-search-on-select="false"
+            :show-labels="false"
+            :selectable="() => versions.length <= 6"
+            placeholder="Filter loader..."
+            :custom-label="(option) => option.charAt(0).toUpperCase() + option.slice(1)"
+          />
+          <multiselect
+            v-model="filterGameVersions"
+            :options="
+              versions
+                .flatMap((value) => value.game_versions)
+                .filter((value, index, self) => self.indexOf(value) === index)
+            "
+            :multiple="true"
+            :searchable="true"
+            :show-no-results="false"
+            :close-on-select="false"
+            :clear-search-on-select="false"
+            :show-labels="false"
+            :selectable="() => versions.length <= 6"
+            placeholder="Filter versions..."
+            :custom-label="(option) => option.charAt(0).toUpperCase() + option.slice(1)"
+          />
+          <multiselect
+            v-model="filterVersions"
+            :options="
+              versions
+                .map((value) => value.version_type)
+                .filter((value, index, self) => self.indexOf(value) === index)
+            "
+            :multiple="true"
+            :searchable="true"
+            :show-no-results="false"
+            :close-on-select="false"
+            :clear-search-on-select="false"
+            :show-labels="false"
+            :selectable="() => versions.length <= 6"
+            placeholder="Filter release channel..."
+            :custom-label="(option) => option.charAt(0).toUpperCase() + option.slice(1)"
+          />
+        </div>
+        <Button
+          class="no-wrap clear-filters"
+          :disabled="
+            filterVersions.length === 0 &&
+            filterLoader.length === 0 &&
+            filterGameVersions.length === 0
+          "
+          :action="clearFilters"
+        >
+          <ClearIcon />
+          Clear filters
+        </Button>
+      </Card>
+      <Pagination
+        v-if="instance.metadata.linked_data"
+        :page="currentPage"
+        :count="Math.ceil(filteredVersions.length / 20)"
+        class="pagination-before"
+        :link-function="(page) => `?page=${page}`"
+        @switch-page="switchPage"
+      />
+      <Card v-if="instance.metadata.linked_data" class="mod-card">
+        <div class="table">
+          <div class="table-row with-columns table-head">
+            <div class="table-cell table-text download-cell" />
+            <div class="name-cell table-cell table-text">Name</div>
+            <div class="table-cell table-text">Supports</div>
+            <div class="table-cell table-text">Stats</div>
+          </div>
```

Comment:
imo there's too much info here. it should just have the first 3 columns (button + name/type, supports info) and be less wide.